### PR TITLE
demonstrate multiple eligibility criteria

### PR DESCRIPTION
**Pull request by crayne**
Only changes to SampleUiManager.java are relevant here.

### CODE REVIEW

File: app/gradle.properties.sample [hunk 1]
```diff
@@ -1,3 +0,0 @@
-# if using Sage Bridge, add these two lines with your study info to gradle.properties
-studyId="STUDY_ID_HERE"
-studyName="STUDY_NAME_HERE"
```
Comment (mikecarroll) [inline app/gradle.properties.sample]: @crayne can you add the `gradle.properties.sample` file back?

**Comment by mikecarroll** [comment]
@crayne Sorry, missed this till now! Can you fix the merge conflicts and return the `.sample` file and I'll merge? Really appreciate this PR!

**Comment by crayne** [comment]
I put the .sample file into my fork.  I'm a newbie at fixing merge conflicts.  Where are they, and can you point me to some info about fixing them?  Thanks.

**Comment by mikecarroll** [comment]
@crayne If you pull the master branch into your fork, it'll tell you where the conflicts are in the code automatically. Should be easy to clean up from there.

**Comment by crayne** [comment]
This should be OK now.

File: build.gradle [hunk 2]
```diff
@@ -22,7 +22,6 @@ allprojects {
     repositories {
         jcenter()
         maven { url "https://jitpack.io" } // for MPAndroidChard, not on jcenter yet
-        mavenLocal()        // for updates to local rstack
```
Comment (mikecarroll) [inline build.gradle]: Maybe we should leave this in?

**Comment by crayne** [comment]
OK, will do

**Comment by mikecarroll** [comment]
👍

**Pull request merged**